state/geth_test.go

```diff
@@ -106,3 +107,25 @@ func TestGethDbReloadData(t *testing.T) {
 		t.Fatalf("Failed to close DB: %v", err)
 	}
 }
+
+func TestGethDb_CreateAccountIsProtected(t *testing.T) {
+	dir := t.TempDir()
+	db, err := MakeGethStateDB(dir, "", common.Hash{}, false, nil)
+	require.NoError(t, err)
+	addr := common.Address{0x22}
+	// First create the account
+	db.CreateAccount(addr)
```

Comment:
Can you assert after CreateAccount() exists after CreateAccount(), e.g., if !db.Exists(addr) { t.Fatalf("Failed to create account") }